spec/lib/catalog/survey_compare_spec.rb

```diff
@@ -36,4 +36,24 @@
       end
     end
   end
+
+  describe "#any_changed?" do
+    context "when the base has changed from topology" do
+      let(:service_plan) { create(:service_plan) }
+      let(:plans) { [service_plan] }
```

Comment:
This is a small nit, but this can be pulled out to be under line 40 and remove line 52.

We should also probably include multiple service plans, since otherwise this is basically the same test as the `changed?`.